src/router/index.ts

```diff
@@ -207,6 +208,40 @@ const router = createRouter({
 			component: TaskDetailView,
 			props: route => ({ taskId: Number(route.params.id as string) }),
 		},
+		{
+			path: '/tasks/create',
+			name: 'task.create',
+			beforeEnter: async (to, from, next) => {
```

Comment:
Please put this logic in a new component.

Reply:
Not sure how that would work, as there is no view for this. The intention is to show the newly created task (resulting in a TaskDetailView) and thus the logic is implemented in the `beforeEnter` handling instead.

Reply:
Can you create a new view which does nothing but contains the logic you have here and then redirects (maybe show a loading spinner etc. while we're at it)? This kind of logic should not go into a beforeEnter handler.